.gitignore

```diff
@@ -1,2 +1,2 @@
 /node_modules
-/dist
\ No newline at end of file
+.eslintcache
```

Comment:
You should keep /dist too!

Reply:
sounds good to me, makes sense that distributables should be kept :)